backend/src/test/java/com/woowacourse/levellog/presentation/LevellogControllerTest.java

```diff
@@ -78,6 +79,31 @@ void save_nameNullOrEmpty_Exception() throws Exception {
             perform.andDo(document("levellog/save/exception-contents"));
         }
 
+        @Test
+        @DisplayName("팀의 인터뷰가 시작된 이후에 요청하는 경우 예외를 던진다.")
+        void save_afterStart_exception() throws Exception {
+            // given
+            final LevellogDto request = LevellogDto.from("content");
+            final String requestContent = objectMapper.writeValueAsString(request);
+            final Long authorId = 1L;
+            final Long teamId = 1L;
+
+            given(jwtTokenProvider.getPayload(ACCESS_TOKEN)).willReturn("1");
+            given(jwtTokenProvider.validateToken(ACCESS_TOKEN)).willReturn(true);
+            doThrow(new InterviewTimeException("인터뷰 시작 전에만 레벨로그 작성이 가능합니다.")).when(levellogService)
+                    .save(request, authorId, teamId);
+
+            // when
+            final ResultActions perform = requestSaveLevellog(teamId, requestContent);
+
+            // then
+            perform.andExpect(status().isBadRequest())
+                    .andExpect(jsonPath("message").value("인터뷰 시작 전에만 레벨로그 작성이 가능합니다."));
+
+            // docs
+            perform.andDo(document("levellog/save/exception-after-start"));
```

Comment:
snippet을 폴더 단위로 생성되게 하는게 좋아보입니다.
```suggestion
            perform.andDo(document("levellog/save/exception/after-start"));
```

Reply:
나중에 이슈 파서 한번에 바꿉시다 😊